### PR TITLE
Add Automated Test

### PR DESCRIPTION
**Pull request by FanchTheSystem**
W.I.P. Please do not merge !

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+$header = <<<EOF
+This file is part of the Blast Project package.
```
Comment (PapsOu) [inline .php_cs]: Still belongs to Blast Project ?

File: bin/ci-scripts/create_travis_database_test.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+# Database creation
+
+###
+### mysql
+###
+
+# (mysql service is started by default by travis for each build instance
+# (mysql travis user is created by travis for each build instance
+mysql -u travis -e 'CREATE DATABASE travis;' -v
```
Comment (PapsOu) [inline bin/ci-scripts/create_travis_database_test.sh]: Perhaps remove mysql usage ?

File: tests/Resources/App/config/config_test_mysql.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+imports:
```
Comment (PapsOu) [inline tests/Resources/App/config/config_test_mysql.yml]: Is this file is really needed because we don't use mysql ?

**Comment by FanchTheSystem** [comment]
https://github.com/libre-informatique/LISemSymfonyProject/pull/7

**Pull request closed without merging**